### ISSUE TITLE
Buff Syndieborgs

### ISSUE THREAD

**Issue by ShadowDeath6** [yogstation13/yogstation-classic#271]
Even with a cost of 80 TC, syndieborgs aren't exactly good. 

LMG is literally a slower firing revolver with a higher ammo capacity. 
Mini ebow got removed
Emag got removed

I'd like for the LMG to get a firing speed boost or let it fire in bursts like the ACTUAL syndie LMG. 
Give it a mini ebow for a decent sidearm while the LMG regains ammo
Give it an emag so it can hack stuff like other borgs. 

If none of these sound reasonable, lower TC cost to 70 or 60 to encourage it actually being used since nobody ever buys it. 

Tl;dr: Shadow gets buttmad at tg for making syndieborgs suck

**Comment by DavidEdwards** [comment]
Everyone, what are the thoughts on this?

**Comment by Time-Green** [comment]
Your buying an expensive murder machine. Not a mad kitten.

They will still fail from ion and flashes.

And perhaps give it a crowbar to open the other borgs and hack them

Still, just walk into AI core and emag him should be enough

**Comment by ShadowDeath6** [comment]
> perhaps give it a crowbar

It has a crowbar still, from when it had an emag, so all it would need is the actual emag to be able to hack other borgs.

**Comment by Time-Green** [comment]
Sorry, never been a syndiborg

**Comment by ghost** [comment]
Giving it an emag'd be nice.

It could also spawn with an inbuilt VTEC module, and perhaps come with a 'syndiborg repair kit'-AKA cyborg repair module?

**Comment by oisin100** [comment]
It doesn't have a VTEC module? Why the fuck doesn't it have a VTEC Module?

Giving it a emag could work well i wonder why they removed it...
Ebow would be an extremely useful sidearm whilst your LMG reloads.
As for it being flashed maybe it could get some sort of "digital light filter" what can be toggled on and off but uses a large amount of energy.

**Comment by X-TheDark** [comment]
I'd at least buff the reload rate of the LMG (currently 1 bullet per 5 seconds) also what's this cyborg repair module?

Also, you know what'd be !FUN!, letting Syndiborgs reload their Grenade Launcher (at the cost of energy, of course, similar to mech weapons).

**Comment by ghost** [comment]
no

flashes and EMP's are their weakness PERIOD

NO CHANGES TO THIS IT IS BALANCED

**Comment by shrimptk** [comment]
Lets finalize gear changes

For a whoppin' 80 TC, you get a syndieborg with:

> VTEC Module 
> Inbuilt Repair Module (300)
> Electromagnetic Sequencer (200)
> Esword (150)
> Burst LMG (200 On Burst)
> Handheld Flashbulb for Disabling other borgs (50)
> Red Crowbar for Opening up other Borgs or emag'd doors (10)
> A Grenade Launcher that must be filled with ammo and costs a lot of energy (500 Per Shot)

Syndies should also have a **Backup Battery** That comes in a little box with the beacon, for when the borg finishes a rampage and gets a wee bit tired.

**Comment by ShadowDeath6** [comment]
Tbh syndieborg doesn't need a self repairer, it can just get welded by one of the 5 newcops, although they're usually useless. 
Also there's a recharger in almost every department, it can probably just use those.

**Comment by ShadowDeath6** [comment]
> Backup Battery

I'd prefer for the borg to have a better cell than making the ops replace it. Not sure what it is now, I think it's 20k, so maybe upgrade to the 35k cell?

**Comment by X-TheDark** [comment]
And it's another damn jury-rigged cell of 25k (manually set) capacity. Higher than that is Hyper-Cap (30k) and Bluespace (40k), but I guess we can jury right it to 35k, it's Syndicate, after all.

**Comment by ShadowDeath6** [comment]
30k would be fine, I thought hyper cap was 35. No need for extra work.

**Comment by shrimptk** [comment]
The thing is, one of the prime times to fight back against such a weapon is while the battery is being replaced, or when it runs out, So increasing the interval of that is kinda lame.

**Comment by ShadowDeath6** [comment]
It can recharge itself at any recharger, battery replacement puts too much work on the incompetent ops.
